src/libraries/Microsoft.Extensions.Logging/src/Properties/TypeForwards.cs

```diff
@@ -0,0 +1,4 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The ref has the type forwards file next to the project file but the src has it in a Properties subfolder. I would move this file out of the Properties folder.

In the past, the Properties folder was associated with the `AppDesignerFolder` property which had a special meaning. I don't know if that's still true these days.

Reply:
Interesting. We are not following that pattern in ref projects so we are inconsistent. Disregard my feedback for this PR. If we feel strongly, we can make this consistent in the future.